tracer/test/Datadog.Trace.TestHelpers/FluentAssertionsExtensions/StringExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿// <copyright file="StringExtensions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using DiffPlex.DiffBuilder;
+using FluentAssertions;
+using FluentAssertions.Execution;
+using FluentAssertions.Primitives;
+
+namespace Datadog.Trace.TestHelpers.FluentAssertionsExtensions;
+
+public static class StringExtensions
+{
+    [CustomAssertion]
+    public static StringAssertions HaveEmptyDiffWhenComparedTo(this StringAssertions value, string expected, string because = "", params object[] becauseArgs)
```

Comment:
nit: I would probably have made this an overload of `Be()` (otherwise I'll probably forget it's there!) but that has disadvantages too 🤷‍♂️

Reply:
I like. Maybe a `bool showDiffOnly` parameter?
```csharp
actualString.Should().Be(expectedString, showDiffOnly: true);
```

Reply:
Done!